apps/sms/views/conversation/js/startup.js

```diff
@@ -72,19 +62,17 @@
   exports.Startup = {
     init() {
       MessageManager.init();
-      Navigation.init();
       ConversationView.init();
-
-      ConversationView.once('visually-loaded', () => {
+      if (Navigation.isDefaultPanel()) {
```

Comment:
__question:__ I admit I haven't read full patch yet, but what does it mean if ```Navigation.isDefaultPanel() === false``` when we load conversation view as a separate document?